pkg/healthcheck/client.go

```diff
@@ -0,0 +1,35 @@
+package healthcheck
+
+import (
+	"context"
+
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-05-01/network"
+
+	acsapi "github.com/openshift/openshift-azure/pkg/api"
+)
+
+type AzureClients struct {
```

Comment:
Use a more specific name like `networkClient`

Reply:
At the moments its only 2 clients, potentially more if we move to separate package. Its generic name?

Reply:
No reason for a centralized struct that holds all clients. I would not want to be importing networking libraries where I don't need them.